haystack_experimental/components/embedders/sentence_transformers_doc_image_embedder.py

```diff
@@ -0,0 +1,335 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import mimetypes
```

Comment:
I wonder if like our converters if we should also make a separate `image_embedders` folder. What do you think?

Reply:
> so LMK if you see good reasons to put them in a separate folder.

Mostly for consistency with `image_converters` and I'm thinking long-term when we add support for other modalities like `Audio` and maybe `Video` that then the `embedders` folder could get crowded  and separating things out would have been nice.

Reply:
done in https://github.com/deepset-ai/haystack-experimental/pull/319/commits/a9068d9c44fd24a2104672b3066733c4a1564220